Shared/DataTransferObjects/AttendanceDto.cs

```diff
@@ -0,0 +1,9 @@
+namespace Shared.DataTransferObjects;
+
+public record AttendanceDto(
+    Guid Id, 
+    Guid EmployeeId, 
+    DateTime CheckInTime, 
+    DateTime? CheckOutTime, 
+    string Status, 
```

Comment:
Consider whether `Status` should be typed using an `enum` instead of a `string`. Using an enum improves type safety and ensures only valid status values are assigned.

**Suggested change**
```suggestion
public enum AttendanceStatus { Present, Absent, Late, OnLeave } // Define this in a shared location

public record AttendanceDto(
    Guid Id,
    Guid EmployeeId,
    DateTime CheckInTime,
    DateTime? CheckOutTime,
    AttendanceStatus Status,
    string? Notes);
```